airgun/widgets.py

```diff
@@ -2143,7 +2143,7 @@ def fill(self, value):
 
 
 class PopOverWidget(Widget):
-    """Popover-content UI widget which contains header, drop_down or input_box and
+    """Popover-content UI widget which contains header, drop_down, input_box, textarea and
```

Comment:
It only contains one of the three and a submit button, right? Not all three and a submit button.
```suggestion
    """Popover-content UI widget which contains header, drop_down, input_box or textarea and
```

Reply:
@lhellebr, It contains one of the three and a submit button.  I'll update to say "header, drop_down, input_box, or textarea and submit button."  Would that ok?